src/main/java/core/basesyntax/FigureArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureArea {
```

Comment:
```suggestion
public interface AreaCalculator {
```
not only figures have areas